core/src/autogluon/core/scheduler/scheduler_constructor.py

```diff
@@ -0,0 +1,131 @@
+import logging
+
+from ..task.base import compile_scheduler_options_v2
+from ..task.base.base_task import schedulers
+from ..utils.utils import setup_compute
+
+logger = logging.getLogger()
+
+
+_scheduler_presets = {
+    'auto': {'searcher': 'random'},
+    # 'grid': {'searcher': 'grid'},  # grid commented out as it isn't compatible with most default model search spaces
+    'random': {'searcher': 'random'},
+    'bayesopt': {'searcher': 'bayesopt'},
+    # 'skopt': {'searcher': 'skopt'},  # TODO: Remove skopt? Is it worthwhile to keep as an option?
```

Comment:
Deciding whether to keep skopt seems like it requires a comparison vs bayesopt? 
skopt offers many additional options for nonstandard Bayesian optimization.

Reply:
I think the major question is if we keep it as a required dependency in core, which it currently is. I'm cautious of this because it is an unmaintained package. I'm happy to keep the support there to explicitly specify it after the user manually installs skopt, but having it as a preset means encouraging it as an option which is recommended/available